src/fragments/lib-v1/auth/android/password_management/10_reset_password_note.mdx

```diff
@@ -0,0 +1,4 @@
+<Callout>
+Note that you must call confirmResetPassword in the same app session as you call resetPassword. If you close the app, you'll need to call resetPassword again.
+As a result, for testing purposes, you'll at least need an input field where you can enter the code sent by the resetPassword api and feed it to confirmResetPassword.
```

Comment:
I feel like this second part is a bit overkill. Why do we need to add it specifically here?

Reply:
I agree and wanted to make this change but this is how it is currently there in V1. I just moved it under Android only as it only pertains to Android. I can remove the last 2 lines if you would like?

Reply:
yes, please! I think that would make that message a lot more concise!

Reply:
Done.